lib/suse_container_urls.pm

```diff
@@ -0,0 +1,76 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Database for URLs of docker images to be tested
+# Maintainer: Fabian Vogt <fvogt@suse.com>
+
+package suse_container_urls;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_sle is_opensuse is_tumbleweed is_leap);
+
+our @EXPORT = qw(
+  get_suse_container_urls
+);
+
+# Returns a tuple of image urls and their matching released "stable" counterpart.
+# If empty, no images available.
+sub get_suse_container_urls {
+    my $version = get_required_var('VERSION');
+    my $dotversion = $version =~ s/-SP/./r; # 15 -> 15, 15-SP1 -> 15.1
+    $dotversion = "${dotversion}.0" if $dotversion !~ /\./; # 15 -> 15.0
+
+    my @image_names  = ();
+    my @stable_names = ();
+    if (is_sle(">=12-SP3")) {
+        my $lowerversion = lc $version;
+        my $nodashversion = $version =~ s/-//r;
+        if (get_var("ARCH") ~~ ["ppc64le", "s390x", "x86_64"]) {
+            push @image_names,  "registry.suse.de/suse/sle-${lowerversion}/docker/update/cr/images/suse/${nodashversion}";
+            push @stable_names, "registry.suse.com/suse/${$nodashversion}";
```

Comment:
```suggestion
            push @stable_names, "registry.suse.com/suse/${nodashversion}";
```
This would otherwise break everything SLE.

Reply:
Unless that causes a syntax error or other kind of fatal runtime exception it won't actually break the test - it'll be silently ignored. I'll fix.

Reply:
Which was the case, see the [log](http://pdostal-server.suse.cz/tests/1009/file/autoinst-log.txt) ... Thank you for fixing it! 😄